docs/explanation/charm-architecture.md

```diff
@@ -39,6 +39,8 @@ The OCI image is built using [Rockcraft](https://canonical-rockcraft.readthedocs
 
 This is done by publishing a resource to Charmhub as described in the [Juju SDK How-to guides](https://juju.is/docs/sdk/publishing).
 
+The OCI image is currently not made to work in an air-gapped environment. You should make sure that it has access to `https://rubygems.org/` (to install gems) and to `https://avatars.discourse.org/` (the official service to create letter avatars).
```

Comment:
Do you think is worth mentioning this document about work offline?
https://juju.is/docs/juju/working-offline

Reply:
I'm not sure it is relevant: the steps described in the juju doc won't help (from what I understand) since it is the container that needs network access.  
If I was adding a section on how to use the charm offline, then I may link to that but here I'm warning that it will not be possible.

Reply:
I see, no worries